docs/content.zh/docs/ops/metrics.md

```diff
@@ -1428,6 +1428,16 @@ Note that the metrics are only available via reporters.
       <td>Maximum recorded duration of a single consecutive period of the task being in the hard back pressure state in the last sampling period. Please check softBackPressuredTimeMsPerSecond and hardBackPressuredTimeMsPerSecond for more information.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>mailboxThroughputPerSecond</td>
+      <td>The number of actions processed from the task's mailbox per second over all actions which includes, e.g., checkpointing, timer, or cancellation actions.</td>
+      <td>Meter</td>
+    </tr>
+    <tr>
+      <td>mailboxThroughputLatencyUs</td>
```

Comment:
`mailboxLatencyUs`? `taskThreadLatencyUs`?

Reply:
Why microseconds? Everything else is either in seconds, milliseconds or nanoseconds.

Reply:
The latency is often below milliseconds and nanoseconds gives large numbers ... I can change it. Would do you prefer? If we need to change it, I would go for milliseconds because below or above zero is still a good metric.

Reply:
Agreed on `mailboxLatencyMS` and `mailboxMailsPerSecond`.